#test error
/* tests break/continue occuring outside of loop body
*/

int main ()
{
	var _ : int ;
	_ = calling_bogus(5, 6) ;
	
	return _;
}

int calling bogus(x: int, y: int) {
	if (x > y)
	{
		break;
		return x;
	}
	else
	{
		return y;
	}
}

